ui.apps/src/main/content/jcr_root/apps/saas-aem-module/i18n/en/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:mix="http://www.jcp.org/jcr/mix/1.0"
```

Comment:
- [ ] Please use the json format to define i18n translations - not a file/folder per entry